### ISSUE TITLE
Javascript error when selecting Logseq as Target Dialect

### ISSUE THREAD

**Issue by johnbrady833** [akosbalasko/yarle#391]
I've installed the x64 version of Yarle 4.10.0 on an Apple M1 MacBook Pro, to import Evernote notes to Logseq.

If I select Logseq as the Target Dialect, I get a Javascript error:

```
Uncaught Exception:
Error: ENOENT, sampleTemplate_logseq.tmpl not found in /Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar
at createError (electron/js2c/asar.js:111:17)
at Object.fs.readFileSync (electron/js2c/asar.js:548:24)
at ElectronStore.<anonymous> (/Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar/dist/ui/main.js:91:43)
at EventEmitter.onChange (/Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar/node_modules/conf/dist/source/index.js:339:22)
at EventEmitter.emit (events.js:223:5)
at ElectronStore.set store [as store] (/Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar/node_modules/conf/dist/source/index.js:297:21)
at ElectronStore.set (/Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar/node_modules/conf/dist/source/index.js:189:20)
at IpcMainImpl.<anonymous> (/Applications/yarle-evernote-to-md.app/Contents/Resources/app.asar/dist/ui/main.js:163:19)
at IpcMainImpl.emit (events.js:223:5)
at WebContents.<anonymous> (electron/js2c/browser_init.js:173:8274)
```

I've looked at FAQs and can't see anything relevant, I hope it's OK to raise as an issue.

**Comment by akosbalasko** [comment]
Hi @johnbrady833 ,

oops, sorry, my bad, i'll fix it soon. it's a so dummy error, I missed to add the default logseq template to the app. 
thanks for raising this up, i hope I'll have time to fix it today.

**Comment by johnbrady833** [comment]
That's amazing - thanks for responding so quickly.

**Comment by akosbalasko** [comment]
Hi @johnbrady833 ,
I released the fix (4.10.1, the latest), and it should work now. Thank you for raising my attention! :) 
I close this issue, but feel free to open an issue if you think that something is still missing!

Cheers,
akos

**Comment by johnbrady833** [comment]
Hi Akos, thanks for the quick fix - that's working great now. Great piece of software, thanks!
